com.unity.visualeffectgraph/Editor/VFXAssetEditorUtility.cs

```diff
@@ -62,7 +62,7 @@ static VisualEffectAssetEditorUtility()
         public const string editorResourcesFolder = "Editor/UIResources";
         public static string editorResourcesPath => VisualEffectGraphPackageInfo.assetPackagePath + "/" + editorResourcesFolder;
 
-        [MenuItem("GameObject/Visual Effects/Visual Effect", false, 10)]
+        [MenuItem("GameObject/VFX/VFX", priority = 12)]
```

Comment:
Should this priority be a const somewhere in the VFX package? Seams to be the only menu entry. If this is the case, don't change it. It will be simpler this way.

Reply:
I think is better to leave it here, as is the only menu of VFX on GameObjects